Makefile

```diff
@@ -32,6 +32,10 @@ ifeq ($(shell expr $(ACTIVE_GO_VERSION_MINOR) \>= 21), 1)
 	LOOPVARFIX := GOEXPERIMENT=loopvar
 endif
 
+# GO_VERSION is the Go version used for the release build, docker files, and
+# GitHub Actions.
+GO_VERSION = 1.22.5
```

Comment:
Should we also mention here that all those versions mentioned are validated against this version? So that this basically is the "reference version"?

Reply:
Good idea.